t/35-script_clone_job.t

```diff
@@ -129,4 +130,60 @@ subtest 'asset download' => sub {
     ok(-f "$temp_assetdir/iso/bar.iso", 'foo touched');
 };
 
+subtest 'clone 2 nodes HA cluster with clone_job' => sub {
+    my %node1_settings = (
+        NAME          => '00007936-sle-15-SP3-Online-x86_64-Build67.1-sles4sap_hana_node01@64bit-sap',
+        TEST          => 'sles4sap_hana_node01',
+        HDD_1         => 'SLE-15-SP3-x86_64-Build67.1-sles4sap-gnome.qcow2',
+        HDDSIZEGB     => 60,
+        PARALLEL_WITH => 'sles4sap_hana_supportserver@64bit-2gbram',
+        NICTYPE       => 'tap',
+        WORKER_CLASS  => 'tap,qemu_x86_64',
+    );
+    my %node2_settings = (
+        NAME          => '00007935-sle-15-SP3-Online-x86_64-Build67.1-sles4sap_hana_node02@64bit-sap',
+        TEST          => 'sles4sap_hana_node02',
+        HDD_1         => 'SLE-15-SP3-x86_64-Build67.1-sles4sap-gnome.qcow2',
+        HDDSIZEGB     => 60,
+        PARALLEL_WITH => 'sles4sap_hana_supportserver@64bit-2gbram',
+        NICTYPE       => 'tap',
+        WORKER_CLASS  => 'tap,qemu_x86_64',
+    );
+    my %supportserver_settings = (
+        NAME         => "00007934-sle-15-SP3-Online-x86_64-Build67.1-sles4sap_hana_supportserver@64bit-2gbram",
+        TEST         => 'sles4sap_hana_supportserver',
+        HDD_1        => 'openqa_support_server_sles12sp3.x86_64.qcow2',
+        HDDSIZEGB    => 60,
+        NICTYPE      => 'tap',
+        WORKER_CLASS => 'tap,qemu_x86_64',
+    );
+    #my $jobid = 7934;
+    #clone_job($jobid, \%options);
+
+    my %node1_job = (
+        settings => %%node1_settings,
+        id => 7936,
+	parents => (
+            Parallel => (7934),
+        ),
+    );
+    my %node2_job = (
+        settings => %%node2_settings,
+        id => 7935,
+	parents => (
+            Parallel => (7934),
+        ),
+    );
+    my %supportserver_job = (
+        settings => %supportserver_settings,
+        id => 7934,
+	children => (
+            Parallel => (7935, 7936),
+        ),
+    );
```

Comment:
I don't think the tidy check is happy with these. It would also make sense to write these hashes in a more compact way.

Reply:
Ok, I will try to compact it as soon as I have a test that works.

Reply:
With the latest change (thanks again!) tidy looks happier.